gdnative/src/lib.rs

```diff
@@ -56,6 +56,8 @@ pub use gdnative_core::*;
 #[doc(inline)]
 pub use gdnative_derive::*;
 
+pub mod prelude;
```

Comment:
I guess we should remove the `use godot_core::*` line before this one (or name specific types to re-export) because it causes `prelude` to appear twice in rustdoc.

Also somehow the `core_types` and `nativescript` modules aren't appearing in rustdoc despite the glob import. Not sure why.

Reply:
This should be resolved now